systemjs/webpack-code-splits/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+
+// See https://github.com/joeldenning/systemjs-webpack-interop
+const systemjsInterop = require('systemjs-webpack-interop')
+
+module.exports = systemjsInterop.modifyWebpackConfig({
+  entry: path.resolve(__dirname, 'src/entry.js'),
+  mode: 'development',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devtool: 'sourcemap',
+  plugins: [new CleanWebpackPlugin()],
```

Comment:
Where is the code to split the bundle?

Reply:
Here: https://github.com/systemjs/systemjs-examples/blob/bf70097688e4c062683d31c9f58399aee6747a49/optimized-builds/webpack-code-splits/src/entry.js#L12